### PR TITLE
Support collect spring-sleuth metrics

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

File: services/gateway-service/build.gradle [hunk 1]
```diff
@@ -37,6 +37,7 @@ ext {
 dependencies {
 	implementation 'org.springframework.cloud:spring-cloud-starter-gateway'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation 'org.apache.skywalking:apm-toolkit-micrometer-registry:8.9.0'
```
Comment (wu-sheng) [inline services/gateway-service/build.gradle:40]:
```suggestion
	implementation 'org.apache.skywalking:apm-toolkit-micrometer-registry:9.1.0'
```

I think 9.1 is a better for showcase(I know there is no actual change)

Comment (mrproliu) [reply services/gateway-service/build.gradle:40]: Also, I see the `songs-service` is using `org.apache.skywalking:apm-toolkit-logback-1.x:8.9.0`, should I need to update this to `9.1.0`?

Comment (wu-sheng) [reply services/gateway-service/build.gradle:40]: Yes, let's bump them up together.

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
You need to update https://github.com/SkyAPMTest/demo-env-installer before redeploying.

**Pull request merged**